indico/web/assets/bundles.py

```diff
@@ -362,16 +362,14 @@ def rjs_bundle(name, *files):
                'Markdown.Editor.js',
                'Markdown.Sanitizer.js'))
 
-base_js = Bundle(jquery, angular, jed, utils, presentation, calendar, indico_jquery, moment,
-                 indico_core, indico_legacy, indico_common, clipboard_js, taggle_js, typewatch_js, fullcalendar_js)
-
 module_js = {
     'bootstrap': rjs_bundle('modules_bootstrap', 'js/indico/modules/bootstrap.js'),
     'cephalopod': rjs_bundle('modules_cephalopod', 'js/indico/modules/cephalopod.js'),
     'categories': rjs_bundle('modules_categories', 'js/indico/modules/categories/display.js'),
     'categories_management': rjs_bundle('modules_categories_management', 'js/indico/modules/categories/management.js'),
     'category_statistics': rjs_bundle('modules_category_statistics', 'js/indico/modules/categories/statistics.js'),
     'vc': rjs_bundle('modules_vc', 'js/indico/modules/vc.js'),
+    'event_creation': rjs_bundle('modules_event_creation', *namespace('js/indico/modules', 'eventcreation.js')),
```

Comment:
I've been grouping them so that `modules` refers to Indico modules. I think if we have more than one JS file for a module it should go into a folder with the name of the Indico module.